Tools/TestWebKitAPI/Tests/WebCore/gstreamer/GStreamerTest.cpp

```diff
@@ -73,6 +73,14 @@ TEST_F(GStreamerTest, gstStructureGetters)
     auto emptyIntOpt = gstStructureGet<int>(structure.get(), "int-val-noexist2"_s);
     if (emptyIntOpt && *emptyIntOpt > 0)
         FAIL() << "emptyIntOpt should be empty, but has value " << *emptyIntOpt;
+
+    GUniquePtr<GstStructure> arrays(gst_structure_new_from_string("bar, empty-array=(GstStructure) <>, struct-array=(GstStructure) <[s1, a=2], [s2, b=3]>"_s));
+    ASSERT_TRUE(gstStructureGetArray<const GstStructure*>(arrays.get(), "empty-array"_s).isEmpty());
+
+    Vector<const GstStructure*> structArray(gstStructureGetArray<const GstStructure*>(arrays.get(), "struct-array"_s));
+    ASSERT_TRUE(gst_structure_is_equal(structArray.at(0), gst_structure_new_from_string("s1, a=2"_s)));
+    ASSERT_TRUE(gst_structure_is_equal(structArray.at(1), gst_structure_new_from_string("s2, b=3"_s)));
```

Comment:
I'm sorry I missed this during the review, those 2 new structures are leaking. Can you follow-up?

Reply:
Fix: https://bugs.webkit.org/show_bug.cgi?id=283906